config/settings.yml

```diff
@@ -234,6 +235,10 @@ modules_appeals_api:
       class: AppealsApi::NoticeOfDisagreementUploadStatusBatch
       description: "Update NoticeOfDisagreement statuses with their Central Mail status"
       every: "60m"
+    appeals_api_sc_cleanup_pii:
```

Comment:
I think you're using an older version of the settings, as this whole `job_schedules` section should have been removed (it was from devops, at least...). This should be added to `sidekiq-schedule.yml` instead.

Reply:
ohhhhhh! ok on it.

Reply:
Heads Up: I'm still seeing the schedules in settings :/